### PR TITLE
atlassian eslint added

### PR DESCRIPTION
**Pull request by adevnadia**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "node": true,
     "browser": true
   },
-  "extends": "eslint:recommended",
+  "extends": "@atlassian/atlassian-fecq",
```
Comment (treshugart) [inline .eslintrc]: We've decided to just use the AirBnb eslint rules in AtlasKit. They're much more stringent and closer to what we want anyways, I think. Maybe use those?

Comment (adevnadia) [inline .eslintrc]: done

**Comment by treshugart** [comment]
lgtm

**Comment by joscha** [comment]
LGTM, I think it would be nice if we could reconsider disabling `prefer-template` later, as I personally think it actually makes things way better.

File: .eslintrc [hunk 2]
```diff
@@ -3,14 +3,18 @@
     "indent": [2, 2],
     "quotes": [2, "single"],
     "linebreak-style": [2, "unix"],
-    "semi": [2, "always"]
+    "semi": [2, "always"],
+    "no-underscore-dangle": 0,
+    "no-param-reassign": 0,
```
Comment (bengummer) [inline .eslintrc:8]: are we ok with reassigning function param variables?

File: .eslintrc [hunk 3]
```diff
@@ -3,14 +3,18 @@
...
+    "consistent-return": 0,
+    "max-len": 0
```
Comment (bengummer) [inline .eslintrc]: better to use a value that allows for long-ish lines (e.g. 120) rather than disabling the rule entirely

**Comment by jpnelson** [comment]
lgtm

**Pull request merged**